scripts/scripts.js

```diff
@@ -442,6 +459,7 @@ export function decorateMain(main) {
   decorateH3(main);
   decorateLinkedPictures(main);
   decorateLinks(main);
+  decorateImages(main);
```

Comment:
I would move this call into the blog article specific autoblocking https://github.com/hlxsites/servicenow/blob/main/scripts/scripts.js#L309-L314

Reply:
Done.